installers/vagrant-dev/radiasoft-download.sh

```diff
@@ -375,6 +376,17 @@ vagrant_dev_provision_private_net() {
 EOF
 }
 
+vagrant_dev_stop_services() {
+    vagrant ssh <<EOF
```

Comment:
I think it would be better to quote the `'EOF'` unless you need interpolation. Same for EOF_BASH